app/controllers/admin/conversations_controller.rb

```diff
@@ -14,11 +15,23 @@ def index
   def show
   end
 
+
+  ## REQUIRE dispatch permissions on the ride zone the conversation is attached to before
+  ## permitting use of close and blacklisting
```

Comment:
This comment mentions protecting close and blacklisting, and I see changes to `close()` like `if… && has_zone_dispatch?`, but I don't see similar changes to the `blacklist` functions. Is that missing or am I misunderstanding?